frontend/components/receipts/orders/cancellation/index.tsx

```diff
@@ -1,9 +1,65 @@
+import { useState, useEffect } from 'react'
 import { ReceiptComponentProps } from '../../receipts'
 import { OrderTable } from '../../utils/order-table'
 import { ReceiptWrapper } from '../../utils/receipt-wrapper'
+import { Intent, Notification } from '@vegaprotocol/ui-toolkit'
+import { OrdersStore, useOrdersStore } from '../../../../stores/orders-store.ts'
+import { useJsonRpcClient } from '../../../../contexts/json-rpc/json-rpc-context.ts'
+
+export const locators = {
+  cancellationNotification: 'cancellation-notification',
+  cancellationView: 'cancellation-view'
+}
+
+export const CancellationNotification = ({ orderId, marketId }: { orderId: string; marketId: string }) => {
+  if (orderId) return null
+
+  return (
+    <div className="mt-2" data-testid={locators.cancellationNotification}>
+      {marketId ? (
+        <Notification intent={Intent.Warning} message={'Cancel all open orders in this market'} />
+      ) : (
+        <Notification intent={Intent.Warning} message={'Cancel all open orders in all markets'} />
+      )}
+    </div>
+  )
+}
+
+export const selectGetOrderById = (state: OrdersStore) => ({ getOrderById: state.getOrderById })
 
 export const CancellationView = ({ cancellation }: { cancellation: any }) => {
-  return <OrderTable {...cancellation} />
+  const { orderId, marketId } = cancellation
+  const [orderDetails, setOrderDetails] = useState<{ order: any; lastUpdated: number | null }>({
+    order: null,
+    lastUpdated: null
+  })
+
+  const { getOrderById } = useOrdersStore(selectGetOrderById)
+  const { request } = useJsonRpcClient()
+
+  useEffect(() => {
+    if (orderId) {
+      getOrderById(orderId, request)
+        .then((result) => {
+          setOrderDetails({ order: result.order, lastUpdated: result.lastUpdated })
+        })
+        .catch((err) => {
+          console.error('Failed to fetch order details:', err)
+        })
```

Comment:
Error handling should be done by the global catch we have. WIth your current setup I think it may be straight up impossible to hit this code branch. Let's kill it.

```suggestion
```